com.microsoft.mrtk.input/Tests/Runtime/Utilities/BaseRuntimeInputTests.cs

```diff
@@ -20,6 +21,12 @@ public abstract class BaseRuntimeInputTests : BaseRuntimeTests
         // Isolates/sandboxes the input system state for each test instance.
         private InputTestFixture input = new InputTestFixture();
 
+        // We only want to isolate the input system state when a test is running in batch mode.
+        // This is indicated by the test either running in the background or explicitly in batch mode.
+        // We do this because some runtime tests utilities rely on keyboard input, and isolating the
+        // input system state means that the phyiscal keyboard is never registered with the application
+        private bool useInputFixture => !Application.isFocused || Application.isBatchMode;
```

Comment:
If you tab away _after_ starting the tests, this probably won't work, right?

Also, this autoprop can change throughout the duration of the test, which would result in the input test fixture being set up and not torn down, or vice versa. If you do this check, you should cache the value once so you don't run into an inconsistent state later on

Reply:
good catch on caching it once rather than letting it auto prop.

If you at tab out after a test has started, it won't run with the fixture. However, setup is run after each test, and the test harness takes a few seconds to start, so I don't think you should encounter much inconsistent behavior unless you are constantly tabbing between the editor and another window.

Reply:
Changed to a new Editor window approach

Reply:
Sweet!